spec/views/content/shared/advisor_promo.html.erb_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+describe "_advisor-promo.html.erb" do
```

Comment:
Small point but we use `adviser` not `advisor` across the site! You're spelling it differently within this spec - I think we should be consistent and use `adviser` @sarahcrack ?